pom.xml

```diff
@@ -304,6 +305,10 @@
                   link="${wildfly.launchdir}/standalone/deployments/${deployment.name}.war"
                   resource="${deployment.parent}/${deployment.name}"
                   overwrite="true" />
+                <copy
+                  tofile="${wildfly.launchdir}/standalone/configuration/optashift-employee-rostering-standalone.xml"
```

Comment:
-1 Veto on customizing the wildfly installation.
Our war file should "just run" on an out-of-the-box installation of wildfly, EAP, wildfly swarm or even weblogic some day.

Reply:
Agreed, but if we are to use System variables, we MUST change an element in the configuration, otherwise build will fail (wildfly plugin does not provide any other way to change that configuration)